docs/CodingConventions.md

```diff
@@ -50,16 +50,14 @@ A list of recommended coding conventions.
    - Local application/library specific imports.
    - Use a blank line between each group of imports.
 
-
-
-## * Why no Unicode for now?
+## * Why do we limit our Unicode use?
 
  - Some characters are visibly indistinguishable. Capital "a" and capital alpha are visibly indistinguishable, but are recognized as separate characters (e.g., search distinguishable).
 
  - Some characters are difficult to read. Sometimes, the overline/overdot/hats overlap with characters making them difficult to see.
 
  - Portability issues. Unicode does not render in Jupyter notebook natively (on OSX).
 
- - If it does improve readability enough, and are not worried about portability, we may introduce a list of permissable characters that are commonly used.
+ - If it does improve readability enough, and are not worried about portability, we may introduce a list of permissible characters that are commonly used.
```

Comment:
Do we apply the conventions across all internals of CLIMA, or just the more public facing bits?

Reply:
My 2 cents: I think anything that improves the readability of internals is "good", while API / user-facing things should be designed with ease of use. So it might make sense to allow arbitrary characters for code that is typed once or twice and read many times, and restrict API code to symbols that can be "easily typed" by a mainstream user.

Reply:
I suggest these rules be followed across all internals. We can put more restrictions on interfaces if necessary, but I think that this list should provide fundamental readability standards.

Reply:
I think we can do both. If you use a unicode function name in a public facing interface, just also have an ascii analogue and document them together
```julia
"""
    ∇(f)
Returns the gradient of some function `f`.
See also: [`gradient`](@ref).
"""
∇(f) = some(complicated(stuff(f)))

"""
    gradient(f)
An alias for [`∇`](@ref).
"""
gradient(f) = ∇(f)
```